lib/convertYardsToMeters.js

```diff
@@ -5,7 +5,9 @@
  */
 
 function convertYardsToMeters(yards) {
-  throw new Error("Not implimented.");
+  if (typeof yards != "number")
+    throw new Error("Invalid Type");
+  return yards * 0.9144;
```

Comment:
could we express the ratio of feet to meters as a fraction to maintain as much precision as JS can afford?

Reply:
Great idea. I'm quite confused though, did you mean **yards** to meters? As far as precision goes, wikipedia says that a yard is 36/39.370113 of a meter. Sound good?